Mathlib/GroupTheory/Divisible.lean

```diff
@@ -81,6 +81,7 @@ Here we adopt a constructive approach where we ask an explicit `div : A → α 
 * `n • div a n = a` for all `n ≠ 0 ∈ α` and `a ∈ A`.
 -/
 class DivisibleBy where
+  /-- The division function. -/
```

Comment:
This is a general comment: I think doc-comments should end with a full stop if and only if they are complete sentences, so
/-- This is a really important consideration about their design. -/
but
/-- my favourite item -/

Can you adjust the new comments accordingly, please? (I'm aware this will be tedious work. Sorry.)
Once you've done so, I will review this promptly.

Reply:
Done.